contest/templates/contest/contestArchive.html

```diff
@@ -0,0 +1,322 @@
+                </td>
+                <td>
+                  <button type="button" class="btn btn-primary">Submit</button>
+                </td>
+              </tr>
+            </table>
+            {% else %}
+            {{ contest.name }}
+            {% endif %}
+          </h2>
+        </div>
+        <div class="modal-body">
+          <ul class="nav nav-tabs" role="tablist" id="infoTab">
+            <li role="presentation" class="active"><a href="#problem" data-toggle="tab">Problem</a>
+            </li>
+            <li role="presentation"><a href="#contestant" data-toggle="tab">Contestant</a>
+            </li>
+            <li role="presentation"><a href="#time" data-toggle="tab">Time</a>
+            </li>
+            <li role="presentation"><a href="#coowner" data-toggle="tab">Coowner</a>
+            </li>
+            <li role="presentation"><a href="#others" data-toggle="tab">Others</a>
+            </li>
+          </ul>
+          <div class="tab-content">
+            <div class="tab-pane active" id="problem">
+              <table class="table" style="text-align:center">
+                <thead>
+                  <tr>
+                    {% if admin %}
+                    <th style="text-align:center">Rejudge</th>
+                    {% endif %}
+                    <th style="text-align:center">#</th>
+                    <th style="text-align:center">Problem</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  {% for problem in contest.problem_list %}
+                  <tr>
+                    {% if admin %}
+                    <td>
+                      <button type="button" class="btn btn-warning btn-xs"> <span class="glyphicon glyphicon-refresh"></span>
+                      </button>
+                      <button type="button" class="btn btn-danger btn-xs"> <span class="glyphicon glyphicon-minus"></span>
+                      </button>
+                    </td>
+                    {% endif %}
+                    <td>{{ problem.id }}</td>
+                    <td>{{ problem.name }}</td>
+                  </tr>
+                  {% endfor %} {% if admin %}
+                  <tr>
+                    <td style="width:10%">
+                      <button type="button" class="btn btn-success btn-xs"> <span class="glyphicon glyphicon-plus"></span>
+                      </button>
+                    </td>
+                    <td></td>
+                    <td style="width:60%">
+                      <input class="form-control">
+                    </td>
+                  </tr>
+                  {% endif %}
+                </tbody>
+              </table>
+            </div>
+            <div class="tab-pane" id="contestant">
+              <table class="table" style="text-align:center">
+                <thead>
+                  <tr>
+                    {% if admin %}
+                    <th></th>
```

Comment:
I mean your \<th\>\</th\> is in if statement, maybe you should add two more spaces in front of it. (so does other {% if ... %} or {% for ... %} tags)